Detour/Source/DetourNavMeshQuery.cpp

```diff
@@ -1827,11 +1829,11 @@ dtStatus dtNavMeshQuery::findStraightPath(const float* startPos, const float* en
 					dtVcopy(portalApex, portalLeft);
 					apexIndex = leftIndex;
 					
-					unsigned char flags = 0;
+					dtStraightPathFlags flags;
 					if (!leftPolyRef)
-						flags = DT_STRAIGHTPATH_END;
+						flags.straightpathEnd = true;
```

Comment:
This line and the similar ones below have also changed meaning; they used to also set all other flags to false. Not sure if it's an issue or not, I'm just pointing it out.